package.json

```diff
@@ -19,7 +19,7 @@
     "jasmine": "^3.5.0"
   },
   "scripts": {
-    "test": "./node_modules/.bin/eslint ./src/main.js ./src/* ./spec/*.js && ./node_modules/.bin/jasmine --random=false",
+    "test": "eslint --fix ./src/main.js ./src/* ./spec/*.js && eslint ./src/main.js ./src/* ./spec/*.js && jasmine --random=false",
```

Comment:
I think it's better to keep the "--fix" argument that already exists on the `lintfix`command. You can access it by `npm run-script lintfix`

Reply:
Okay, let me try it out!